bahmni-lite/docker-compose.yml

```diff
@@ -397,6 +397,17 @@ services:
         - 'bahmni-queued-reports:/mounts/bahmni-queued-reports'
         - 'configuration_checksums:/mounts/configuration_checksums'
 
+  ipd:
```

Comment:
@Arjun-Go @binduak I feel that we should not bundle IPD container with Bahmni Lite as we are not shipping any configuration related to IPD in clinic-config and also not the ADT nor bed-management module is enabled.